ui/apps/platform/src/routePaths.ts

```diff
@@ -263,6 +265,10 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
     'platform-cves': {
         resourceAccessRequirements: everyResource(['Cluster']),
     },
+    'platform-workload-cves': {
+        featureFlagRequirements: allEnabled(['ROX_PLATFORM_CVE_SPLIT']),
+        resourceAccessRequirements: everyResource(['Deployment', 'Image']),
```

Comment:
Related to the comment about grouping keys, the super simple split architecture assumes same `resourceAccessRequirements` for both routes, correct?

Especially if they remain separated, how about a comment?

Reply:
Yes, as it is right now both pages are the same code and have the same resource requirements. This might change in the future, but there are no plans from what I am aware of.